src/rust/plugin-work-queue/src/client.rs

```diff
@@ -0,0 +1,108 @@
+#![allow(warnings)]
```

Comment:
Any reason to leave this in? Since this is a completely new file, we should be able to eliminate any warnings up front.

If it's because we've got a bunch of essentially stubbed-out implementations and there are a bunch of legitimate warnings, would it be possible to explicitly ignore those specific warnings at the site? That would presumably make it easier to remove them once the implementations are filled in. I know that's potentially a lot of annotation, but these blanket exceptions at the top of the file are really easy to forget about, and can mask a lot of other things that we legitimately don't want to allow.

Reply:
It should be totally removed, it was just for dev purposes.